### DISCUSSION TITLE
How to set Kong must be accessed through domain name instead of ip form？

### DISCUSSION THREAD

**Discussion by dream-mo** [Kong/kong#8335] [Help]
When I want to configure external client to access ports 8000 and 8443 of Kong, they cannot be accessed by ip, but must be accessed through a certain domain name. But I searched all over the kong.conf configuration file, but I can't seem to find any relevant configuration items. Usually I will configure it like this in nginx:

```nginx
server {
  listen 80 default;
  server_name _;
  return 403;      # Access is forbidden without setting server_name
}
```
Can someone who knows please help? thanks

**Comment by locao** [comment]
The full answer will depend on how you installed Kong. Anyway, to get this to work you have to add the `server` section you mentioned to the enf of the `templates/nginx_kong.lua` file, before the line containing `]]`.
